roles/collectd/README.rst

```diff
@@ -0,0 +1,76 @@
+Collectd
+========
+
+Collectd role for deploying Collectd.
+
+This role has the following global settings:
+
+.. data ::  Hostname  
+
+   Hostname to append to metrics                    
+   
+   Default: {{ ansible_hostname }}
```

Comment:
this should be surrounded in double backticks to make it a code literal. Like:

    ``{{ blah }}``